pkg/utils/status.go

```diff
@@ -273,11 +279,17 @@ func calculateState(ctx context.Context, instance *csmv1.ContainerStorageModule,
 func SetStatus(ctx context.Context, r ReconcileCSM, instance *csmv1.ContainerStorageModule, newStatus *csmv1.ContainerStorageModuleStatus) {
 
 	log := logger.GetLogger(ctx)
-	instance.GetCSMStatus().State = newStatus.State
-	log.Infow("Driver State", "Controller",
-		newStatus.ControllerStatus, "Node", newStatus.NodeStatus)
-	instance.GetCSMStatus().ControllerStatus = newStatus.ControllerStatus
-	instance.GetCSMStatus().NodeStatus = newStatus.NodeStatus
+	if instance.GetName() == "application-mobility" {
```

Comment:
even though appmob is the only module to have csm state, wouldn't it be better to have a separate function to set status for appmob ?

Reply:
if there are more modules i was thinking of having a different function.
That if for future update